siac/rentercmd.go

```diff
@@ -110,21 +118,81 @@ func abs(path string) string {
 	return abspath
 }
 
-func renterdownloadqueuecmd() {
+func renteruploadscmd() {
```

Comment:
it's more of a pedantic thing. All functions are supposed to have docstrings, including test functions. It helps with things like grep, and helps you orient yourself if you aren't familiar with the project as a whole.

A lot of these functions were created before we were strict with how we do things. Siac is also heroically undertested.